rf-ng/src/app/article-list/article-list.component.ts

```diff
@@ -5,9 +5,7 @@ import { IPageInfo } from 'ngx-virtual-scroller';
 import { Observable, Subscription ,  BehaviorSubject, interval } from "rxjs";
 import * as moment from 'moment';
 import { scan, map, switchMap, startWith } from "rxjs/operators";
-
-
-
+import { NgxMasonryModule } from 'ngx-masonry';
```

Comment:
you have to import this in the app.module and add it to the list of module imports.